readme.md

```diff
@@ -153,91 +150,99 @@ button.on( "click", handleClickEvent );
 
-This might not be your first time hearing it, and definitely won't be your last. A callback is a piece of executable code that is passed as an argument to other code, which is expected to invoke (or "call back") that executable code at some convenient time.
+This might not be your first time hearing it and definitely won't be your last. A callback is a piece of executable code that is passed as an argument to other code, which is expected to invoke (or "call back") that executable code at some convenient time.
+
+In other words, **a callback contains code that will be run at a later time.**
 
 The invocation may be immediate or it might happen later. In the example above, `handleClickEvent` is our callback. The invocation happens when the button is clicked.  
 
 > We can also pass in anonymous functions (i.e., define a nameless function directly inside of the event listener) instead of previously-defined functions.
 
-### You Do: Practice (2:55 - 3:05, 10 minutes)
+### You Do: Practice (10 minutes / 0:35)
 
-Visit this [repository](https://github.com/ga-wdi-exercises/event-listener-practice.git) and follow the instructions.
+Visit this [repository](https://github.com/ga-wdi-exercises/event-listener-practice) and follow the instructions.
 
-### Before We Go On... (3:05 - 3:10, 5 minutes)
+### Callbacks: Calling vs. Referencing (5 minutes / 0:40)
 
-Usually when we do anything with functions, we put parentheses after the function name. Here, we have `handleClickEvent` without any parens.
+When we call a function, we place parentheses after the function name. In the case of callbacks like `handleClickEvent`, however, there are no parens.
 
-Try adding parentheses at the end of this line:
+Try adding parentheses at the end of this line...
 
 ```js
-button.on("click", handleClickEvent() );
+button.on("click", handleClickEvent());
 ```
 
-Let's refresh our pages and then discuss the result. What was different? Why?
+<details>
 
-> You'll notice that "I was clicked!" pops up immediately upon reload. Also note that the event while it does fire, isn't doing anything. When we include `()` we invoke the function expression. Without the `()`, we're using the function expression as a reference.
+  <summary><strong>Refresh the page. What happened? What was different? Why?</strong></summary>
 
+  > "I was clicked!" pops up immediately upon reload. When we include `()` we invoke the function expression immediately, not after the user interacts with the page. Without the `()`, we're using the function expression as a reference.
 
-## Break (3:10 - 3:20, 10 minutes)
+</details>
 
-### You Do: Color Scheme Switcher (3:20 - 3:50, 30 minutes)
 
-Clone this repo and follow the readme instructions: **[Color Scheme Switcher](https://github.com/ga-dc/color-scheme-switcher)**.
+## Break (10 minutes / 0:50)
 
+### You Do: [Color Scheme Switcher](https://github.com/ga-dc/color-scheme-switcher) (20 minutes / 1:10)
```

Comment:
jsut fyi i think this was given to them during interview for WDI, right? Alternate exercise could be that stoplight one